README.md

```diff
@@ -119,23 +119,6 @@ You also must set the viewer for this kind of media.
   _Collabora Online Preview_.
 - Click _Save_.
 
-### User permissions
-
-There are three levels of permissions. _Administrator_,
-_Collaborator_, and _Viewer_, in order of most privileges to the
-least. Each lesser privilege is included in the higher level.
-_Administrator_ includes _Collaborator_, and _Collaborator_ includes
-_Viewer_.
-
-By default, the user role `administrator` is mapped to the Collabora
-Online administrator. This allow accessing the console.
-
-The user role `authenticated` is the default permission for
-collaboration, i.e. edit a document with Collabora Online.
-
-The user role `anonymous` by default disallows editing and is the
-minimal permission for viewing documents in Collabora Online.
-
```

Comment:
We can rethink this section. For now I just removed it to avoid confusion.

Reply:
Right now it is unclear how these new permissions work. Is it possible to have a short write up here?

Reply:
I re-added a section, I hope it is more clear this way.

The "which can be managed at `/admin/people/permissions`." should be obvious to people who are familiar with Drupal, but I mention it anyway.